### DISCUSSION TITLE
FORMATO 12 / 24 HORAS AM PM

### DISCUSSION THREAD

**Discussion by Piero15** [Makuna/Rtc#129] [Q&A] [answered]
Hola la descripción del modulo indica que es compatible con el formato tanto de 12 como 24 hora cambiando el Am al PM quisiera saber como configurar ese apartado usando esta libreria gracias

**Comment by Makuna** [comment, marked as answer]
This library exposes time in the 24 hour model only.  I made this decision to reduce code size for translation to/from the chips when it was often not needed.  

Its easy enough to translate this to AM and PM on the client sketch if needed when displaying to or taking from the user.  Since it is already a requirement for the sketch to manage time zones, the minor extra work to manage AM/PM felt like the right place to put that work.

p.s.  I always suggest the model that the core time time/date be ZULU time (search for time zone offset zero).  This is what the module should use and also what the sketch should store and only translate when displaying to the user if needed.

**Reply by Piero15** [reply]
Muchas gracias makuna hare un video recomendando tu libreria es tan simple y tan bueno super genial
